lib/Dialect/TTNN/Pipelines/TTNNPipelines.cpp

```diff
@@ -60,6 +54,8 @@ void createTTNNPipelineAnalysisPasses(
 
 void createTTNNPipelineLoweringPasses(
     OpPassManager &pm, const TTIRToTTNNBackendPipelineOptions &options) {
+  // Add pass to add layout information.
```

Comment:
Looks strange - as if we are mixing TTIR and TTNN by saying this creates TTNN layout prior to lowering, but it actually does not introduce ops from TTNN dialect?

Reply:
I've added the comment on PR, but basically TTNNOps verify methods and TTIRToTTNN conversion relies heavily on layout attribute being present.
And on top of that we will remove ToLayout pass at some point when we add OpConstraintsInterface, so this should be in between phase.